src/realm/impl/transact_log.hpp

```diff
@@ -81,6 +81,8 @@ enum Instruction {
     instr_LinkListNullify = 36, // Remove an entry from a link list due to linked row being erased
     instr_LinkListClear = 37,   // Ramove all entries from a link list
     instr_LinkListSetAll = 38,  // Assign to link list entry
+    instr_AddSubtableSearchIndex = 39,  // Add search index to subtable
+    instr_RemoveSubtableSearchIndex = 40   // Remove search index from subtable
```

Comment:
It's a bit tricky. Reusing the instruction would need add_search_index() to be available either on a subtable Table or subtable Descriptor (since replicator can select one of such). Table would be bad since a table is not always available (if the root table has no rows). Making the method available on Descriptor would mismatch the syntax for things like adding columns to a subtable, which is through the root table, at least that's the way our unit tests do it (even though it is possible to do though a descriptor too). So all solutions seem like a tradeoff... but I'm open to input